KMNavigationBarTransition/UINavigationController+KMNavigationBarTransition.m

```diff
@@ -49,6 +49,10 @@ + (void)load {
     });
 }
 
+- (UIColor *)km_containerViewBackgroundColor {
```

Comment:
I would have to override it in each controller... I really need this one so please can you merge it? I will shouw you hwo it look like when you have transparent backgrohnd in main view controller... Each backgroud is white insted of transparent...